.github/workflows/release.yaml

```diff
@@ -0,0 +1,38 @@
+name: Build and Release
+
+on:
+  workflow_dispatch:
+
+inputs:
+  version:
+    required: true
+    description: "Tag to apply to release commit"
+
+jobs:
+  Build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - name: Install dependencies
+        uses: bahmutov/npm-install@v1
```

Comment:
Is this dependency doing anything more sophisticated than running `npm install`? If not, it might not be worth including as a dependency. Eh, it seems fine, I don't feel strongly about it.

Reply:
It has npm caching built-in which seemed like an easy win.